addons/web/static/src/views/fields/float_time/float_time_field.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.FloatTimeField" owl="1">
         <span t-if="props.readonly" t-esc="formattedValue" />
-        <input t-else="" t-att-id="props.id" t-att-type="props.inputType" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" inputmode="decimal" class="o_input" />
+        <input t-else="" t-att-id="props.id" t-att-type="props.inputType" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" class="o_input" />
```

Comment:
the absence of attribute should be tested. This is the kind of thing we easily re-introduce when refactoring the code

Reply:
done